compiler/testData/codegen/box/delegatedProperty/delegateToGenericJavaProperty.kt

```diff
@@ -0,0 +1,24 @@
+// TARGET_BACKEND: JVM
+// WITH_REFLECT
+// WITH_RUNTIME
+// FILE: J.java
+public interface J<T> {
+    public T getValue();
+}
+
+// FILE: box.kt
+class Impl(val x: String) : J<String> {
+    override fun getValue() = x
+}
+
+val j1: J<String> = Impl("O")
+// Note that taking a reference to `J<T>::value` is not permitted by the frontend
```

Comment:
NB this actually looks like a front-end bug. Not in the scope this PR, anyway.

Reply:
Filed https://youtrack.jetbrains.com/issue/KT-47299

Reply:
@pyos ^ please add a reference to KT-47299 into this comment